gematria/datasets/pipelines/benchmark_cpu_scheduler.py

```diff
@@ -141,3 +142,19 @@ def verify(self):
     cpu_mask = list(os.sched_getaffinity(0))
     if self._cpu_mask != cpu_mask:
       raise ValueError('Expected the CPU mask to not change.')
+
+
+class BenchmarkSchedulerImplementations(Enum):
+  NoScheduling = 1
+  Default = 2
```

Comment:
Nit: In Python, enum value names should follow the naming for constants (`BIG_SNAKE_CASE`).